resources/helm/overlays/istio-telemetry/prometheus/values.yaml

```diff
@@ -53,4 +53,16 @@ prometheus:
   # When this option is set as false, this certificate provisioning mechanism is disabled.
   provisionPrometheusCert: true
 
+global:
```

Comment:
are these necessary if global.yaml is configuring them?

Reply:
We previously removed merging `global.yaml` file logic, but now we added it back so it's safe to remove it from promotes values.